packages/desktop/src/webview.rs

```diff
@@ -110,6 +110,7 @@ impl WebviewEdits {
                     data.into_any()
                 }
             }
+            dioxus_html::EventData::Visible(_) => data.into_any(),
```

Comment:
This looks the same as the branch following it. Should it be merged or do we need to handle converting visible data to any in a different way?

Reply:
Indeed. I'm going to remove this branch.